### PR TITLE
[API] Update UPGRADE-1.8 file after adding ApiBundle

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by Zales0123** [comment]
Thanks, Grzegorz! :1st_place_medal:

File: UPGRADE-1.8.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    +               id: sylius.admin_user_provider.email_or_name_based
+            
+            firewalls:
+    +           new_api:
+    +               pattern: "%sylius.security.new_api_admin_regex%/.*"
+    +               stateless: true
+    +               anonymous: true
+    +               provider: sylius_admin_api_user_provider
+    +               json_login:
+    +                   check_path: /new-api/authentication-token
```
Comment (stloyd) [inline UPGRADE-1.8.md]: Shouldn't this be also with regexp?
```
check_path: "%sylius.security.new_api_admin_regex%/authentication-token"

Comment (lchrusciel) [reply UPGRADE-1.8.md]: It should. Would you like to open a PR?

Comment (stloyd) [reply UPGRADE-1.8.md]: Done in: https://github.com/Sylius/Sylius/pull/11293

**Pull request merged**